packages/accounts/src/msca/types.ts

```diff
@@ -16,7 +16,7 @@ export interface IMSCA<
 
   extendWithPluginMethods: <AD, PD>(
     plugin: Plugin<AD, PD>
-  ) => IMSCA<TTransport, TProviderDecorators & PD> & AD;
+  ) => this & IMSCA<TTransport, TProviderDecorators & PD> & AD;
```

Comment:
is `this` necessary? why?

Reply:
yea, so before if you didn't include it then if you previously extended the account you would lose them on the second extend.

If you try it out on the `main` branch right now, you can see that happening in the multi owner builder function. Once you extend with TokenReceiver, the methods for MultiOwner no longer resolve